Documentation/Issue-Guide.md

```diff
@@ -0,0 +1,38 @@
+### Overview
+This page outlines how the WCF team thinks about and handles issues.  In general, we follow the guidance given at [CoreFx Issue Guidelines](https://github.com/dotnet/corefx/blob/master/Documentation/issue-guide.md).
+
+Add new issues or track existing ones at [Issues](https://github.com/dotnet/wcf/issues).
+
+### Labels
+We use GitHub labels on our issues in order to classify them.  We have the following categories per issue:
+* **Area**: These labels call out the assembly or assemblies the issue applies to. In addition to tags per assembly, we have a few other tags: Infrastructure, for issues that relate to our build or test infrastructure, and Meta for issues that deal with the repository itself, the direction of the .NET Core Platform, our processes, etc.
```

Comment:
I was originally thinking we would link back to CoreFx's issues-guide.md for the bulk of this and talk only about our differences, but I appreciate the value in having live links to the queries in our repo.  Our additions to the basic CoreFx labels are:
 * "feature request" is a label we assign to issues that are requests for new features. This is intended as a way to tag issues where we want active community discussion on the importance of a proposed new features
 * "not yet supported" is a label we assign to issues for features that work in Windows Store but throw PlatformNotSupportedException from the libraries in this repo.  Most of these are under development and are expected to be fixed.
 * "stress" is a label we assign for issues discovered when running in stress labs or heavily loaded applications
 * "test bug" is a label we assign to **all** issues related to testing. We label these further with "infrastructure" (for test infrastructure changes) or "enhancement" (for missing tests or those that need enhancement).